### PR TITLE
Git_Basics.rst: Correct formatting for git status

### PR DESCRIPTION
**Pull request by aman9875**
This fixes the formatting for "git status"
under pushing the commit section.

Closes https://github.com/coala/coala/issues/5061

**For short term contributors:** we understand that getting your commits well
defined like we require is a hard task and takes some learning. If you
look to help without wanting to contribute long term there's no need
for you to learn this. Just drop us a message and we'll take care of brushing
up your stuff for merge!

### Checklist

- [x] I read the [commit guidelines](http://coala.io/commit) and I've followed
      them.
- [x] I ran coala over my code locally. (*All commits have to pass
      individually.* It is not sufficient to have "fixup commits" on your PR,
      our bot will still report the issues for the previous commit.) You will
      likely receive a lot of bot comments and build failures if coala does not
      pass on every single commit!

After you submit your pull request, **DO NOT click the 'Update Branch' button.**
When asked for a rebase, consult [coala.io/rebase](https://coala.io/rebase)
instead.

Please consider helping us by reviewing other peoples pull requests as well:

- pick up any PR at <https://coala.io/review>
- review it (check <https://coala.io/reviewing> for more info)
- if you are sure that it needs work, use `cobot mark wip <URL>` to get it out
  of the review queue.

The more you review, the more your score will grow at coala.io and we will
review your PRs faster!

### CODE REVIEW

File: docs/Developers/Git_Basics.rst [hunk 1]
```diff
@@ -236,7 +236,8 @@ Before you push the commit, ensure that you are not developing on master again
 by running:
 
 ::
-    git status
+
+   $ git status
```
Comment (Makman2) [inline docs/Developers/Git_Basics.rst]: 2 spaces like everywhere else in the document.

However, this changes the formatting? The `::` actually define the code-style-formatting here^^

**Comment by Makman2** [comment]
Typo in commit body:
`formattiing`

;)

**Comment by Makman2** [comment]
unack 9a5e08ad753ad9a88016e1143c43b0da649f5380

**Comment by aman9875** [comment]
@Makman2  Made the changes.

**Comment by Makman2** [comment]
Nope still the same ;)

**Comment by aman9875** [comment]
@Makman2 is it fine now?

**Comment by vibhanshuvaibhav** [comment]
LGTM

File: docs/Developers/Git_Basics.rst [hunk 2]
```diff
@@ -236,7 +236,8 @@ Before you push the commit, ensure that you are not developing on master again
 by running:
 
 ::
-    git status
+
+    $ git status
```
Comment (Makman2) [inline docs/Developers/Git_Basics.rst]: Be consistent, 2 spaces indentation (I've told you that here last time ;) )

**Comment by Makman2** [comment]
unack 413c1cf0dc894e6dd31f76043c9f00e78947322b

File: docs/Developers/Git_Basics.rst (hunk 2 above)
Comment (aman9875) [reply docs/Developers/Git_Basics.rst]: But the rest of the document (except line 230) uses 4 spaces for indentation

Comment (Makman2) [reply docs/Developers/Git_Basics.rst]: Oh indeed, sorry my bad :)

**Comment by Makman2** [comment]
ack 413c1cf

**Comment by Makman2** [comment]
@gitmate-bot rebase

**Comment by Makman2** [comment]
@gitmate-bot ff

**Pull request merged**